src/components/Quote.vue

```diff
@@ -279,6 +289,13 @@ export default {
 			align-items: center;
 			gap: 4px;
 			color: var(--color-text-maxcontrast);
+
+			&-edit,
+			&-server {
+				white-space: nowrap;
+				overflow: hidden;
+				text-overflow: ellipsis;
```

Comment:
still, not ellipsized 
![image](https://github.com/user-attachments/assets/d47fa70f-b801-49ce-adb3-086b3eccb8c1)

add `width: 100%` to quote__main